GCTC-NTGC/gc-digital-talent
# Use v1 Conditions of Employment on MVP pool candidate forms

Resolves #3387 

To test:
- [ ] Set the FEATURE_APPLICANTSEARCH flag to false and recreate your PHP container
- [ ] Run refresh scripts
- [ ] If necessary, run data seeders with `docker-compose exec -w /var/www/html/api php sh -c "php artisan db:seed"`
- [ ] In one tab, open the search page (http://localhost:8000/en/talent/search) and find the Conditions of Employment options
- [ ] In another tab, get to the Create Pool Candidate form:
    1. Go to http://localhost:8000/en/admin/pools
    2. Select "View Candidates" for a pool
    3. Click "Create Pool Candidate" in top right corner
- [ ] In a third tab, get to the Update Pool Candidate form:
    1. Go to http://localhost:8000/en/admin/pools
    2. Select "View Candidates" for a pool
    3. Select "Edit" for a pool candidate
- [ ] For both forms, the Operational Requirements should show the same options with the same text as on the Search page. The text should match in both English and French. They should be the options shown in https://github.com/GCTC-NTGC/gc-digital-talent/pull/3357.
